proto/destination.proto

```diff
@@ -164,6 +164,12 @@ message Route {
   // outstanding request will be cancelled, a timeout error response will be
   // returned, and no more retries will be attempted.
   google.protobuf.Duration timeout = 5;
+
+  // If this list is non-empty, traffic to this service should instead be split
+  // between the backends in this list.  Each backend should receive a portion
+  // of the traffic proportional to its weight.  If this list is empty, traffic
+  // should be sent to this service as normal.
+  repeated Backend traffic_split = 6;
```

Comment:
I'm not a fan of the word "backend" here-- it's neither specific nor clear. and the phrase "traffic split", while it reflects the SMI naming, introduces a new term, traffic, into this API.